docs/other-software/crystal.md

```diff
@@ -0,0 +1,53 @@
+# CRYSTAL
+
+CRYSTAL is a general-purpose program for the study of crystalline solids. The
+CRYSTAL program computes the electronic structure of periodic systems within
+Hartree Fock, density functional or various hybrid approximations (global,
+range-separated and double-hybrids). The Bloch functions of the periodic
+systems are expanded as linear combinations of atom centred Gaussian
+functions. Powerful screening techniques are used to exploit real space
+locality. Restricted (Closed Shell) and Unrestricted (Spin-polarized)
+calculations can be performed with all-electron and valence-only basis sets
+with effective core pseudo-potentials. The current release is CRYSTAL17.
+
+## Useful Links
+
+- [CRYSTAL home site](https://www.crystal.unito.it)
+- [CRYSTAL tutorials](http://tutorials.crystalsolutions.eu)
+- [CRYSTAL licensing](http://www.crystalsolutions.eu")
+
+## Using CRYSTAL on ARCHER2
+
+CRYSTAL is only available to users who have a valid CRYSTAL license. You 
+request access through SAFE:
+
+- [How to request access to package groups](https://epcced.github.io/safe-docs/safe-for-users/#how-to-request-access-to-a-package-group)
+
+Please have your license details to hand.
+
+## Running parallel CRYSTAL jobs
+
+The following script will run a CRYSTAL job using 256 cores (2 nodes). it
+assumes that the input file is tio2.d12
+
+```slurm
+#!/bin/bash
+#SBATCH -n 256
```

Comment:
I'd use `--ntasks` to keep consistency and thus use the long option parameter names.